daisy_workflows/build-publish/windows_media/windows-install-media.wf.json

```diff
@@ -0,0 +1,91 @@
+{
+  "Name": "windows-install-media",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "media_disk": "media-disk",
+    "build_date": "${TIMESTAMP}",
+    "image_prefix": "windows-install-media",
+    "gcs_path": {"Value": "gs://gce-image-archive/windows-install-media", "Description": "GCS path to export image to"},
```

Comment:
"gcs_url": {
      "Required": true,
      "Description": "The GCS url that the image raw file exported to."
    },

Reply:
done